### PR TITLE
Added ZAI stablecoin on various L2s 

### PR DESCRIPTION
**Pull request by senamakel**
We have deployed ZAI into various L2s with native restaking (similar to Renzo). Added the double counted flag to the addresses as the staked collateral minted on L2s gets restaked on eth mainnet.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/mahaxyz exports TVL: 
        
 
 
    ethereum-pool2            161.67 k
    pool2                     161.67 k
    ethereum                  83.97 k
    arbitrum                  2
    scroll                    1
    xlayer                    1
    base                      1
    optimism                  1
    blast                     1
    bsc                       1
    linea                     1
    
    total                    83.98 k

**Comment by llamatester** [comment]
The adapter at projects/mahaxyz exports TVL: 
        
 
 
    ethereum-pool2            161.67 k
    pool2                     161.67 k
    ethereum                  83.97 k
    arbitrum                  2
    blast                     1
    xlayer                    1
    scroll                    1
    optimism                  1
    base                      1
    bsc                       1
    linea                     1
    
    total                    83.97 k

**Comment by llamatester** [comment]
The adapter at projects/mahaxyz exports TVL: 
        
 
 
    ethereum-pool2            161.67 k
    pool2                     161.67 k
    ethereum                  83.97 k
    base                      3
    arbitrum                  2
    bsc                       1
    optimism                  1
    scroll                    1
    blast                     1
    xlayer                    1
    linea                     1
    
    total                    83.98 k

**Comment by g1nt0ki** [comment]
ZAI tvl = tokens backing zai, which is only on ethereum right? marking it doublecounting doesnt change it.

**Comment by g1nt0ki** [comment]
Sorry, I am tempted to reject this because more projects that brige and go multichain will do the same thing. Also, ZAI has restaking? how does that work?

**Comment by senamakel** [comment]
> ZAI tvl = tokens backing zai, which is only on ethereum right? marking it doublecounting doesnt change it.

Yes, however users can also mint ZAI by staking it on L2s as well. The bridged collateral gets sent to mainnet but the tokens are stay on the L2. https://docs.maha.xyz/mechanics/native-re-staking Similar to renzo.

> Sorry, I am tempted to reject this because more projects that brige and go multichain will do the same thing. Also, ZAI has restaking? how does that work?

Restaking not in the terms of LRT restaking; But basically USDC on L2s can be staked on L2s on that gets sent to mainnet. https://docs.renzoprotocol.com/docs/integrations/l2-native-restaking. I guess a more accurate explaination would be to call it native L2 minting.

Basically we're trying to show our presence on L2s through the USDC captured and used to natively mint ZAI on these L2s. Hence the call to `totalSupply(..)`

**Comment by senamakel** [comment]
Would be open to ideas or better ways to represent this 🙏

**Comment by g1nt0ki** [comment]
hi @senamakel I had a discussion regarding this with my team, we believe showing renzo (and kelp dao) as multichain was a mistake and added it tot my todo to revert them to shown only on ethereum.  Closing this PR for now.
